GLMakie/assets/shader/distance_shape.frag

```diff
@@ -41,7 +41,7 @@ flat in vec4            f_uv_texture_bbox;
 // These versions of aastep assume that `dist` is a signed distance function
 // which has been scaled to be in units of pixels.
 float aastep(float threshold1, float dist) {
-    return min(1.0, f_viewport_from_u_scale)*smoothstep(threshold1-ANTIALIAS_RADIUS, threshold1+ANTIALIAS_RADIUS, dist);
```

Comment:
I should probably check if that changed anything. For text it was irrelevant, but maybe it matters for scatter?

Reply:
I don't see a difference in characters scatters. I guess it's fine